hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/schedule/ScheduleSignSpecs.java

```diff
@@ -691,27 +724,29 @@ public HapiSpec overlappingKeysTreatedAsExpected() {
                         getAccountBalance(ADDRESS_BOOK_CONTROL).hasTinyBars(changeFromSnapshot("before", +2)));
     }
 
+    @HapiTest
     public HapiSpec retestsActivationOnSignWithEmptySigMap() {
         return defaultHapiSpec("RetestsActivationOnCreateWithEmptySigMap")
                 .given(newKeyNamed("a"), newKeyNamed("b"), newKeyListNamed("ab", List.of("a", "b")), newKeyNamed(ADMIN))
                 .when(
-                        cryptoCreate(SENDER).key("ab").balance(667L),
+                        cryptoCreate(SENDER).key("ab").balance(665L),
```

Comment:
Those numbers should not be changed.

Reply:
They're just random-ish values; they aren't significant...